linkis-dist/package/db/linkis_dml.sql

```diff
@@ -562,8 +562,9 @@ INSERT INTO `linkis_ps_dm_datasource_type_key` (`data_source_type_id`, `key`, `n
 INSERT INTO `linkis_ps_dm_datasource_type_key` (`data_source_type_id`, `key`, `name`, `name_en`, `default_value`, `value_type`, `scope`, `require`, `description`, `description_en`, `value_regex`, `ref_id`, `ref_value`, `data_source`, `update_time`, `create_time`) VALUES (@data_source_type_id, 'password', '密码(Password)', 'Password', NULL, 'PASSWORD', NULL, 1, '密码(Password)', 'Password', '', NULL, '', NULL, now(), now());
 INSERT INTO `linkis_ps_dm_datasource_type_key` (`data_source_type_id`, `key`, `name`, `name_en`, `default_value`, `value_type`, `scope`, `require`, `description`, `description_en`, `value_regex`, `ref_id`, `ref_value`, `data_source`, `update_time`, `create_time`) VALUES (@data_source_type_id, 'elasticUrls', 'ES连接URL(Elastic Url)', 'Elastic Url', NULL, 'TEXT', NULL, 1, 'ES连接URL(Elastic Url)', 'Elastic Url', '', NULL, '', NULL, now(), now());
 
-
+-- https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-reference-jdbc-url-format.html
 select @data_source_type_id := id from `linkis_ps_dm_datasource_type` where `name` = 'mysql';
+INSERT INTO `linkis_ps_dm_datasource_type_key` (`data_source_type_id`, `key`, `name`, `name_en`, `default_value`, `value_type`, `scope`, `require`, `description`, `description_en`, `value_regex`, `ref_id`, `ref_value`, `data_source`, `update_time`, `create_time`) VALUES (@data_source_type_id, 'address', '地址', 'address', NULL, 'TEXT', NULL, 0, '地址(host1:port1,host2:port2...)', 'address(host1:port1,host2:port2...)', NULL, NULL, NULL, NULL,  now(), now());
```

Comment:
about name_en 
It is recommended to capitalize the first letter  
```
'地址', 'address' ->`地址', 'Address' 

'地址(host1:port1,host2:port2...)', 'address(host1:port1,host2:port2...)'
-->
-> 
'地址(host1:port1,host2:port2...)  多地址模式。和主机名+端口单地址模式，配置一种即可', 'Address (host1:port1,host2:port2...) multi-address mode, and host name + port single-address mode, just configure one'
```

Reply:
need update upgrade sql filea at linkis-dist/package/db/upgrade/1.3.2_schema/mysql

Reply:
OK